commit 8c61b900ebb8ec5918ffd776ba1a61a5f022566d
Author: Thomas Gleixner <tglx@linutronix.de>
Date:   Wed Jan 30 13:30:16 2008 +0100

    x86: make early_indentify_cpu static
    
    early_indentify_cpu is only used in setup_64.c
    
    Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
    Signed-off-by: Ingo Molnar <mingo@elte.hu>

diff --git a/arch/x86/kernel/setup_64.c b/arch/x86/kernel/setup_64.c
index 97a4976..0e2bffe 100644
--- a/arch/x86/kernel/setup_64.c
+++ b/arch/x86/kernel/setup_64.c
@@ -141,6 +141,8 @@ struct resource bss_resource = {
 	.flags = IORESOURCE_RAM,
 };
 
+static void __cpuinit early_identify_cpu(struct cpuinfo_x86 *c);
+
 #ifdef CONFIG_PROC_VMCORE
 /* elfcorehdr= specifies the location of elf core header
  * stored by the crashed kernel. This option will be passed
@@ -844,7 +846,7 @@ struct cpu_model_info {
 /* Do some early cpuid on the boot CPU to get some parameter that are
    needed before check_bugs. Everything advanced is in identify_cpu
    below. */
-void __cpuinit early_identify_cpu(struct cpuinfo_x86 *c)
+static void __cpuinit early_identify_cpu(struct cpuinfo_x86 *c)
 {
 	u32 tfms;
 
diff --git a/include/asm-x86/proto.h b/include/asm-x86/proto.h
index 64fe816..9e3d51f 100644
--- a/include/asm-x86/proto.h
+++ b/include/asm-x86/proto.h
@@ -40,8 +40,6 @@ extern int nohpet;
 
 extern void early_printk(const char *fmt, ...) __attribute__((format(printf,1,2)));
 
-extern void early_identify_cpu(struct cpuinfo_x86 *c);
-
 extern int k8_scan_nodes(unsigned long start, unsigned long end);
 
 extern void numa_initmem_init(unsigned long start_pfn, unsigned long end_pfn);
